### PR TITLE
Creates cropped image component

### PR DESCRIPTION
**Pull request by clayton-duarte**
- Includes new CroppedImage component;
- Includes more test cases to various components to raise overall test coverage;
- Bumps up `jest.coverageThreshold` config;

### CODE REVIEW

File: src/components/Modal/modals.doc.tsx [hunk 1]
```diff
@@ -1,18 +1,22 @@
 import React from 'react'
-import Modal from '.'
+
```
Comment (l0rdcafe) [inline src/components/Modal/modals.doc.tsx:2]: question: any reason for the .doc.tsx extension?

Comment (clayton-duarte) [reply src/components/Modal/modals.doc.tsx:2]: just to identify if the component is a real component or just something we built to make the documentation easier

File: src/components/Select/index.spec.tsx [hunk 2]
```diff
@@ -59,30 +50,70 @@ describe('Select', () => {
...
   it('should render when passed the checkout variant', () => {
-    const mockedOptions = [...Array(5)].map((a, index) => ({
-      value: `value-${index}`,
-      label: `label-${index}`,
-      key: `key-${index}`,
-    }))
+    const { container } = render(
+      <ThemeProvider>
+        <Select
+          onChange={mockOnChange as Dispatch<string>}
```
Comment (l0rdcafe) [inline src/components/Select/index.spec.tsx:56]: question: did we need the cast? can we cast to `jest.Mocked<Dispatch<string>>`?

Comment (clayton-duarte) [reply src/components/Select/index.spec.tsx:56]: good point man! lemme test that

File: src/components/Modal/modals.doc.tsx (hunk 1 above)
Comment (l0rdcafe) [reply src/components/Modal/modals.doc.tsx:2]: okay, in this case why is `Checkbox.tsx`?? maybe i'm confused heh

Comment (clayton-duarte) [reply src/components/Modal/modals.doc.tsx:2]: `Checkbox.tsx`? where?

File: src/components/Select/index.spec.tsx (hunk 2 above)
Comment (clayton-duarte) [reply src/components/Select/index.spec.tsx]: worked!!!

File: src/components/Modal/modals.doc.tsx (hunk 1 above)
Comment (l0rdcafe) [reply src/components/Modal/modals.doc.tsx:2]: `src/components/CheckBox/index.tsx`

Comment (clayton-duarte) [reply src/components/Modal/modals.doc.tsx:2]: `src/components/CheckBox/index.tsx` is a real component, like `src/components/Modal/index.tsx` also is. But `src/components/Modal/modal.doc.tsx` is a documentation helper, not a component

Comment (l0rdcafe) [reply src/components/Modal/modals.doc.tsx:2]: ahhhh okay i see now, my bad man thanks for explaining

**Review by l0rdcafe — APPROVED** [review]

**Pull request merged**